.buildkite/pipeline.full.yml

```diff
@@ -133,7 +97,7 @@ steps:
     commands:
       - scripts/ci-run-macos-tests.sh
 
-  - label: Run MacOS e2e tests for Unity 2021
+  - label: Run macOS e2e tests for Unity 2021
```

Comment:
Will changing this label mean that the Github branch protection checks need updating?

Reply:
I think we're ok as the status checks all seem to be lower-cased anyway.